src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const properties = sourceString.split(';').map((prop) => prop.split(':'));
+
+  for (const property of properties) {
+    if (property.length < 2) {
+      continue;
+    }
+
+    const prop = property[0].trim();
+    const value = property[1].trim();
+
+    result[prop] = value;
+  }
```

Comment:
[CODE PRACTICE] - Avoid using loops in this task, use iteration methods instead. Consider using `forEach` to iterate over the properties since you are not returning a new array and just modifying the `result` object.